tpl/hunger.twig

```diff
@@ -2,23 +2,42 @@
     <h1>Hunger | <a href="http://tum.sexy" class="logo">TUM.<strong>sexy</strong></a></h1>
 </div>
 
-<div class="container">
-    <p>This is the 'Speiseplan' of the current week in the FMI Bistro of the Informatik Fakultät at TUM.</p>
-    <p>
-        {% if pdfImage %}
-            <img src="{{ pdfImage }}" class="img-responsive" style="max-width: 100%;"/>
-        {% elseif pdfLink %}
-            <a href="{{ pdfLink }}" target="_blank">Original menu as PDF</a>
+
+
+<p>This is the 'Speiseplan' of the current week in the FMI Bistro of the Informatik Fakultät at TUM as well as the Mensa Garching.</p>
+
+<div class="row" style="margin-top:10%">
+    <div class="one-half column">
+        <h3>FMI Bistro</h3>
+
+        {% if fmiBistroWeek == null %}
+            <p>An error occured while fetching the FMI Bistro menu.</p>
         {% else %}
-            No menu for today 😭 <br/>
-            Maybe check the <a href="{{ linkPage }}">Bistros page</a> and <a href="https://github.com/mammuth/TUM.sexy/issues">open a bug report</a> or redirect all complaints to <a
-                href="mailto:info@wilhelm-gastronomie.de">the Bistro</a>.
+            {% for day in fmiBistroWeek %}
+                <strong>{{day.getDate() | date("l, F j")}}<sup>{{day.getDate() | date("S")}}</sup></strong>
```

Comment:
From a usability perspective: I don't think anyone cares about previous days...

Only showing days >= today would probably be more useful 🎉

Reply:
Yep, on the old version (with the previous bistro owner) we only showed all future days of the current week 👍

Reply:
done